network-config-analyzer/K8sPolicyYamlParser.py

```diff
@@ -422,14 +422,25 @@ def parse_policy(self):
         res_policy.selected_peers &= self.peer_container.get_namespace_pods(self.namespace)
 
         ingress_rules = policy_spec.get('ingress', [])
+        exception = ''
         if ingress_rules:
             for ingress_rule in ingress_rules:
-                res_policy.add_ingress_rule(self.parse_ingress_rule(ingress_rule, res_policy.selected_peers))
+                try:
+                    res_policy.add_ingress_rule(self.parse_ingress_rule(ingress_rule, res_policy.selected_peers))
```

Comment:
Better have this try-catch block inside `parse_ip_block`, so you don't have to repeat it for ingress and for egress.

Reply:
Done